From d4ddc6b9d5f9477550b627def0399a5eb6b0b7e2 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Fri, 28 Feb 2014 14:20:08 +0100
Subject: [PATCH] builder: add libguestfs.gpg to EXTRA_DIST

It needs to be in EXTRA_DIST, otherwise it is not put in the dist.

(cherry picked from commit 962837650cfb95e50d3c44578cbf47f7ec625271)
---
 builder/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/builder/Makefile.am b/builder/Makefile.am
index f0cb1dd..2dc9d23 100644
--- a/builder/Makefile.am
+++ b/builder/Makefile.am
@@ -27,6 +27,7 @@ AM_CFLAGS = \
 
 EXTRA_DIST = \
 	$(SOURCES) \
+	libguestfs.gpg \
 	virt-builder.pod \
 	virt-index-validate.pod \
 	test-virt-builder.sh \
-- 
1.8.5.3

